go.mod

```diff
@@ -3,20 +3,41 @@ module github.com/casdoor/oss
 go 1.18
 
 require (
+	cloud.google.com/go/storage v1.31.0
 	github.com/Azure/azure-storage-blob-go v0.15.0
 	github.com/aliyun/aliyun-oss-go-sdk v2.2.2+incompatible
 	github.com/aws/aws-sdk-go v1.44.4
+	github.com/aws/aws-sdk-go-v2 v1.22.1
```

Comment:
Why a lot of AWS SDKs are added here?

Reply:
Cloudflare provides a demonstration of the use of the SDK. I'll delete it and try to see if it works.

Reply:
Cloudflare and AWS are two different companies. We don't need to add AWS SDK to use Cloudflare services.

If the tests require the dependencies, we can remove the tests

Reply:
cloudflare R2 implements S3 API
https://developers.cloudflare.com/r2/api/s3/api/

The current calling method provided by r2 is to use the aws sdk
https://developers.cloudflare.com/r2/examples/aws/aws-sdk-go/

In addition, aws-sdk-go-v2 is a newer sdk version. Do I need to downgrade to v1 version?
https://aws.github.io/aws-sdk-go-v2/docs/migrating/